assets/notifications.js

```diff
@@ -204,14 +205,17 @@ var Notifications = (function(options) {
      * @returns {jQuery|HTMLElement|*}
      */
     this.renderRow = function (object) {
-        var keywords = ['id', 'title', 'description', 'url', 'type'];
-        var ret, html = self.opts.listItemTemplate;
-
-        html = '<div class="notification notification-{type} ' +
-            (object.seen ? 'notification-seen' : 'notification-unseen') +
-            '" data-route="{url}" data-id="{id}">' +
-                html +
-                '</div>';
+        var keywords = ['title', 'description', 'url'];
+        var ret, html;
+
+         html = $(self.opts.tag, {
+            class: "notification notification-" + object['type'] + " "+(object.seen ? 'notification-seen' : 'notification-unseen'),
+            "data-route":object['url'],
```

Comment:
please use the `object.url` form, with an extra space before 'object'
single quotes instead of double quotes

Reply:
Ok, i don´t know why i used brackets in that object. Is a mistake